src/aavruti/FieldValidations.java

```diff
@@ -0,0 +1,90 @@
+package aavruti;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class FieldValidations {
+
+	WebDriver driver = new ChromeDriver();
+
+	//Enter values in all the fields
+	void enterValuesInFields() {
+		driver.get("G:\\SeleniumVideos\\Assignment1\\SeleniumAssignment_1.html");
+		if(driver.getTitle().equals("TECHNOCREDITS")) {
+			System.out.println("Selenium Assignment opened successfully");
+		}
+		else {
+			System.out.println("Browser not launched successfully");
+		}
+
+		WebElement firstName = driver.findElement(By.id("first name"));
```

Comment:
You should only store any value in webElement when you want to perform more than 1 action. 
But here you are only performing one action which sendKeys() so you should write like this:
driver.findElement(By.id("first name")).sendKeys("Aavruti");